File: app2_java_jokenpo_app/app/src/main/java/com/estudoparajava/pedrapapeloutesoura/MainActivity.java
package com.estudoparajava.pedrapapeloutesoura;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;


import java.util.Random;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void selecionadoPedra(View view){
        this.opcaoSelecionada("pedra");
    }

    public void selecionadoPapel(View view){
        this.opcaoSelecionada("papel");
    }

    public void selecionadoTesoura(View view){
        this.opcaoSelecionada("tesoura");
    }

    public void opcaoSelecionada(String opcaoSelecionada){
        ImageView imageView = findViewById(R.id.imageView);
        TextView textView = findViewById(R.id.textView3);
        int numero;
        numero = new Random().nextInt(3);
        String[] opcoes = {"pedra" , "papel", "tesoura"};
        String opcaoApp = opcoes[numero];

        switch (opcaoApp){
            case "pedra" : imageView.setImageResource(R.drawable.pedra);
                break;
            case "papel" : imageView.setImageResource(R.drawable.papel);
                break;
            case "tesoura" : imageView.setImageResource(R.drawable.tesoura);
                break;
            default: imageView.setImageResource(R.drawable.padrao);
        }
        if(
                (opcaoApp.equals("tesoura") && opcaoSelecionada.equals("papel")) ||
                (opcaoApp.equals("papel") && opcaoSelecionada.equals("pedra")) ||
                (opcaoApp.equals("pedra") && opcaoSelecionada.equals("tesoura"))
        ){
                textView.setText("App é o vencedor");
        }else if(
                (opcaoSelecionada.equals("tesoura") && opcaoApp.equals("papel")) ||
                (opcaoSelecionada.equals("papel") && opcaoApp.equals("pedra")) ||
                (opcaoSelecionada.equals("pedra") && opcaoApp.equals("tesoura"))
        ){
                textView.setText("Você é o vencedor");
        }else{
            textView.setText("Empate");
        }
    }


}

File: Kotlin/app1_frases_do_dia/app/src/main/java/com/estudoparajava/frasesdodia/MainActivity.kt
package com.estudoparajava.frasesdodia

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.TextureView
import android.view.View
import android.widget.TextView
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    lateinit var gerarTexto: TextView
    var texto = arrayOf(
        "O importante não é vencer todos os dias, mas lutar sempre.",
        "Maior que a tristeza de não haver vencido é a vergonha de não lutado.",
        "É melhor conquistar a si mesmo do que vencer mil batalhas.",
        "Enquanto houver vontade de lutar haverá esperança de vencer.",
    )


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun gerarFrase(view: View){
        this.gerarTexto = findViewById<TextView>(R.id.textView)
        gerarTexto.text = texto[Random.nextInt(4)]

    }
}